arangod/RestHandler/RestStatusHandler.cpp

```diff
@@ -56,6 +56,10 @@ RestStatus RestStatusHandler::execute() {
   result.add("server", VPackValue("arango"));
   result.add("version", VPackValue(ARANGODB_VERSION));
 
+#if defined(TRI_HAVE_POSIX_THREADS)
+  result.add("pid", VPackValue(getpid()));
```

Comment:
Can we simply use `Thread::currentProcessId()` here?
Then we can also remove the ifdef.

Reply:
I don't mind that unless it is the PID of the arangod instance fine with me. This is used for `top` and needs to be specifically the PID of the instance.

Reply:
Yes, that's what `Thread::currentProcessId()` will do. Except that it will work on Windows too, in case we want to add arango inspector support for it later.